scripts/biomedical/NCBI_Assembly/Summary_Report/scripts/test.sh

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Author: Sanika Prasad
+# Date: 2024/05/31
+
+#!/bin/bash
+
+# Command
+python3 scripts/process_test.py
```

Comment:
Add the following lines so that the dc java tool to generate the json report is downloaded and run:
# download data commons java test tool version 0.1-alpha.1k
mkdir -p tmp; cd tmp
wget https://github.com/datacommonsorg/import/releases/download/0.1-alpha.1k/datacommons-import-tool-0.1-alpha.1-jar-with-dependencies.jar
cd ..

# run java test tool
java -jar tmp/datacommons-import-tool-0.1-alpha.1-jar-with-dependencies.jar lint tMCFs/ncbi_assembly_summary_report.tmcf output_file/ncbi_assembly_summary.csv *.mcf
mv dc_generated ncbi_assembly_report

Reply:
Please note that this tool does generate a  "MCF_MalformedComplexValueParts" error for infraspecific_name such as userMessage": "Complex value must have 2 (e.g., [Years 10]) or 3 (e.g., [Years 10 20]) components :: value: '[P11H11F7]', components: 1, property: 'infraspecificName', node: 'dcid:bio/GCA_006436775.1'". However, in this case the text value with a single thing contained within the brackets is expected and right as it is a text value and not a Quantity or Range like the json tool test is expecting.

Reply:
This is also the case for the  "MCF_QuantityRangeMalformedValues" error thrown:   
"userMessage": "Malformed start component in QuantityRange value; must be a number or '-' :: value: 'Cocollar', property: 'biologicalIsolate', node: 'dcid:bio/GCA_003029025.2'",
    "counterKey": "MCF_QuantityRangeMalformedValues"
   
And the MCF_QuantityMalformedValue error:
"userMessage": "Quantity value must be a number :: value: '[Brazil:Minas Gerais-Bicas2:1999]', property: 'biologicalIsolate', node: 'dcid:bio/GCA_000865405.1'",
    "counterKey": "MCF_QuantityMalformedValue"
    
    
Ignore all of these errors